### DISCUSSION TITLE
Enable an amiibo to brush multiple weapons

### DISCUSSION THREAD

**Discussion by awsl2546** [jordanbtucker/totk-no-loot-boxes#18] [Ideas]
一个amiibo可以刷出多种武器，但使用目前的mod只能获取固定的武器。能否增加一些其他武器的掉落？

An amiibo can brush multiple weapons, but using the current mod can only obtain fixed weapons. Can you increase the drop of some other weapons?

**Comment by jordanbtucker** [comment]
I'm not sure what you mean by fixed weapons? What other weapons would you like to see?

**Reply by awsl2546** [reply]
例如“Breath of the Wild_Zelda”可以获得“Royal Shield”或“Radiant Shield”，但在装载这个mod之后只能获得“Royal Shield”，“Radiant Shield”就无法获得了。

For example, "Breath of the Wild_Zelda"amiibo can obtain "Royal Shield" or "Radiant Shield", but after loading this mod, only "Royal Shield" can be obtained, and "Radiant Shield" cannot be obtained.

**Comment by jordanbtucker** [comment]
Thanks for clarifying. Why do you want items with worse stats?

**Reply by awsl2546** [reply]
抱歉，我研究了一下，有些武器我认为更好，实际上更差，例如上次提到的“Royal Shield”和“Radiant Shield”。打扰了。感谢你提供的mod。

Sorry, I have studied some weapons that I think are better, but in reality, they are even worse, such as the "Royal Shield" and "Radiant Shield" mentioned last time. Excuse me. Thank you for providing the mod.

**Reply by jordanbtucker** [reply]
Thanks. I haven't studied the weapons. Which weapons should each amiibo drop?
